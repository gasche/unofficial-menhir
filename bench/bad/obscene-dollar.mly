%token <int> A
%token B C

%%

main: x = A; y = B
  { $2 + $1 }

