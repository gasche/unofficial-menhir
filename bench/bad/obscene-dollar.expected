File "obscene-dollar.mly", line 7, characters 9-11:
Error: please do not say: $1. Instead, say: x.
